llama/generation.py

```diff
@@ -13,6 +15,33 @@ class LLaMA:
     def __init__(self, model: Transformer, tokenizer: Tokenizer):
         self.model = model
         self.tokenizer = tokenizer
+        self._generate_one_token_fn = self._generate_one_token
+        xm._init_ordinal_world_size()
```

Comment:
There is no need for this line for the new whl.

Reply:
Removed this line